modules/local/cluster_file/main.nf

```diff
@@ -0,0 +1,36 @@
+process CLUSTER_FILE {
+    tag "Create cluster file for GAS call"
+    label 'process_single'
```

Comment:
I think this is similar but slightly different to the module in #11. Maybe the header generation stuff? Not sure if it'll cause a conflict later when one of them is merged.

Reply:
Since PR #11 hasn't been merged in yet, this PR contains some of the code from that PR, which should disappear once #11 is merged into `dev` (and `dev` is merged into this branch).

Reply:
Thanks Aaron 😄 I will be sure to merge https://github.com/phac-nml/gasnomenclature/pull/11 to `dev` first and reverse merge `dev` back here.